src/cijoe/cli/cli.py

```diff
@@ -83,6 +84,29 @@ def cli_integrity_check(args):
     return 0
 
 
+def cli_script(args):
+    """Create python script with the boiler-plate code necessary for running cijoe as a standalone script."""
+    filename = "cijoe-script.py"
+    if os.path.exists(filename):
+        sys.exit("Error: File 'cijoe-script.py' already exists.")
+    
+    lines = [
+        """from cijoe.cli.cli import cli_interface""",
+        "",
+        """def main(args, cijoe, step):""",
+        """    err, state = cijoe.run("echo 'Hello World!'")""",
+        """    if "Hello" not in state.output():""",
+        """        return 1""",
+        """    return err""",
+        "",
+        """if __name__ == "__main__":""",
+        """    cli_interface(__file__)""",
+    ]
+
+    with open(filename, "x") as script:
+        script.write("\n".join(lines))
```

Comment:
How about doing something similar to `cli_example()`, i.e. having an example script in `src/cijoe/core/scripts` and copying that. That way you don't have to write the code as strings.
Also that would allow the documentation to just point to that script when describing the output.

Reply:
Good idea @karlowich !

Reply:
@safl can you check that this is the "right" way to include it in this way?

Reply:
> @safl can you check that this is the "right" way to include it in this way in the documentation?

The documentation needs an overhaul, so just getting the information in there is great, so, LGTM!